hedera-mirror-rest/api/v1/openapi.yml

```
@@ -1314,20 +1322,26 @@ components:
                     type: string
     EvmAddress:
       type: string
-      description: A network entity encoded as an EVM address in hex
+      description: A network entity encoded as an EVM address in hex.
+      format: binary
+      pattern: '^(\d{0,9}\.){0,2}[A-Fa-f0-9]{40}$'
+      example: "0000000000000000000000000000000000001f41"
+    EvmAddressOutput:
+      type: string
+      description: A network entity encoded as an EVM address in hex.
       format: binary
       minLength: 42
```

Just noticed this was wrong before, if you can adjust.
```suggestion
      minLength: 40
```

---

Done.